app/cli/cmd/workflow_workflow_run_list.go

```diff
@@ -70,6 +71,7 @@ func newWorkflowWorkflowRunListCmd() *cobra.Command {
 
 	cmd.Flags().StringVar(&workflowID, "workflow", "", "workflow ID")
 	cmd.Flags().BoolVar(&full, "full", false, "full report")
+	cmd.Flags().StringVar(&status, "status", "", fmt.Sprintf("filter by workflow run status: %v", listAvailableWorkflowStatusFlag()))
```

Comment:
how does %v looks like in the output? Sometimes I remember using %q to get a quoted list. 

It would be great for future patches that you add the result in the description of the PR just for reference to reviewers.

Reply:
I've updated the description of the PR, it basically prints the values on its default format.

```
$chainloop wf workflow-run ls --help
List workflow runs

Usage:
  chainloop workflow workflow-run list [flags]

Aliases:
  list, ls

Flags:
      --full              full report
  -h, --help              help for list
      --limit int         number of items to show (default 20)
      --next string       cursor to load the next page
      --status string     filter by workflow run status: [FAILED EXPIRED CANCELLED INITIALIZED SUCCEEDED]
      --workflow string   workflow ID

Global Flags:
      --artifact-cas string    URL for the Artifacts Content Addressable Storage (CAS) (default "api.cas.chainloop.dev:443")
  -c, --config string          Path to an existing config file (default is $HOME/.config/chainloop/config.toml)
      --control-plane string   URL for the Control Plane API (default "api.cp.chainloop.dev:443")
      --debug                  Enable debug/verbose logging mode
  -i, --insecure               Skip TLS transport during connection to the control plane
  -o, --output string          Output format, valid options are json and table (default "table")
  -t, --token string           API token. NOTE: Alternatively use the env variable CHAINLOOP_API_TOKEN

Process finished with the exit code 0

```